contents/blog/yc-2-years-on.md

```diff
@@ -0,0 +1,83 @@
+
+The number of partners that can deliver relevant advice to scaling is lower (whereas the majority are world experts in finding product market fit) - scaling PostHog is quite different to scaling [supersonic air flight](https://boomsupersonic.com/). However, there *are* many visiting partners that will meet with you - they bring deeper operating experience.
+
+We've reduced how often we feel the need to do office hours a lot. During the batch, it was without exception every 2 weeks. Now, it's every couple of months. I think this is because post product market fit, it has felt easier to know what to build - we have many users to talk to and a clear [ideal customer profile](../handbook/strategy/strategy#target-customers-for-2022). This means we usually know what we need to do, and simply doing the work as fast as we can.
+
+However, when we do need help - it's pretty leveraged. For example, we recently confirmed that the majority of our growth comes from word of mouth. We wanted to talk to someone relevant about this - YC meant we could talk to Nicolas Dessaigne (Algolia cofounder) about how he'd approach the problem. This had a major impact on how we think about marketing. As a result, PostHog's marketing team objective is literally "accelerate word of mouth growth" today.
+
+## Money...
+
+YC has a Continuity Fund. This can invest in later stage companies.
+
+We've been fortunate to partner with them. They've:
+
+* invested roughly $20M
+* helped us stay focused on (the right) users in board meetings
+* provided endless further intros to people for advice and hiring as needed
+
+This isn't guaranteed though, but it wouldn't have happened if we'd not gone through the batch.
+
+## ... and how to spend it (hiring)
+
+[Work at a Startup](https://www.workatastartup.com/) is provided to YC companies for during and after the batch.
+
+This has given us a bunch of interviews with candidates.
+
+## Stonks / angel investing side hustle
+
+One of the things that surprised me was how common it is for founders to invest in other founders. Firstly, how come they've all got the cash?! It turns out that some founders act as scouts for VC funds, some have had good tech jobs before, and others have sold some stock. Being part of YC means you have easy access to lots of promising startups.
+
+I did this twice then stopped primarily because I felt it'd get distracting to do it a lot. My personal life [got a bit chaotic](ceo-diary-3) last year.
+
+## Conclusion
+
+There's a lot of ongoing support. A lot of this is provided through network effects so it scales surprisingly well.
+
+We've both given and received advice from dozens (probably 50 or more) of other companies. This has given us years of experience in weeks, but it can make it tempting to stop thinking for yourself. I've been suggested in a [full team feedback session](../handbook/people/feedback#full-team-feedback-sessions) that I should work from first principles more often, which was helpful advice.
```

Comment:
```suggestion
We've both given and received advice from dozens (probably 50 or more) other companies. This has given us years of experience in weeks, but it can make it tempting to stop thinking for yourself. I've been suggested in a [full team feedback session](../handbook/people/feedback#full-team-feedback-sessions) that I should work from first principles more often, which was helpful advice.
```